lib/sw-precache.js

```diff
@@ -183,7 +183,7 @@ function generate(params, callback) {
     if (params.runtimeCaching) {
       var pathToSWToolbox = require.resolve('sw-toolbox/sw-toolbox.js');
       swToolboxCode = fs.readFileSync(pathToSWToolbox, 'utf8')
-        .replace('//# sourceMappingURL=sw-toolbox.map.json', '');
+        .replace('//# sourceMappingURL=./build/sw-toolbox.map.json', '');
```

Comment:
Yeah that'll be the case. I'll raise a bug.

On Fri, 10 Jun 2016, 19:23 Jeffrey Posnick, <notifications@github.com>
wrote:

> In lib/sw-precache.js
> <https://github.com/GoogleChrome/sw-precache/pull/131#discussion_r66659215>
> :
>
> > @@ -183,7 +183,7 @@ function generate(params, callback) {
> >      if (params.runtimeCaching) {
> >        var pathToSWToolbox = require.resolve('sw-toolbox/sw-toolbox.js');
> >        swToolboxCode = fs.readFileSync(pathToSWToolbox, 'utf8')
> > -        .replace('//# sourceMappingURL=sw-toolbox.map.json', '');
> > +        .replace('//# sourceMappingURL=./build/sw-toolbox.map.json', '');
>
> The version of sw-toolbox.js that gets installed via npm currently has
> the line //# sourceMappingURL=./build/sw-toolbox.map.json in it.
>
> If that's not intentional then it may need to be updated in a sw-toolbox
> 3.2.2.
>
> —
> You are receiving this because you were assigned.
>
>
> Reply to this email directly, view it on GitHub
> <https://github.com/GoogleChrome/sw-precache/pull/131/files/320c709b19231052d64728cc045f956771d08581#r66659215>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAIh8EYzZkDgkPLQaQUHxXgCm7rdrUa5ks5qKauggaJpZM4IzL2u>
> .
>